blueocean-github-pipeline/src/main/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubScmSaveFileRequest.java

```diff
@@ -83,7 +82,7 @@ public Object save(@Nonnull String apiUrl, @Nullable String owner, @Nullable Str
             if(!StringUtils.isBlank(sha)){
                 body.put("sha", sha);
             }
-            final Map ghResp = HttpRequest.put(String.format("%s/repos/%s/%s/contents/%s",
+            final Map<?,?> ghResp = HttpRequest.put(String.format("%s/repos/%s/%s/contents/%s",
```

Comment:
I don't like to see warnings all the time... probably just an Eclipse thing, maybe I should change my settings...